cmd/secrets-store-csi-driver/main.go

```diff
@@ -112,7 +112,7 @@ func main() {
 
 	mgr, err := ctrl.NewManager(cfg, ctrl.Options{
 		Scheme:             scheme,
-		MetricsBindAddress: *metricsAddr,
+		MetricsBindAddress: fmt.Sprintf(":%d", *metricsAddr),
```

Comment:
this loses the ability to choose how the metrics service should be bound (whether just local interface or all interfaces)

Reply:
@tam7t Could you please elaborate?